system/doc/reference_manual/data_types.xml

```diff
@@ -79,6 +79,25 @@
 11> <input>1_234.333_333</input>
 1234.333333
 </pre>
+<marker id="float_represenation_problem" /><note>
```

Comment:
Maybe you want to correct the spelling of `representation'.

Reply:
Thanks for seeing this. Fixed: 855323f